docs/coreos.md

```diff
@@ -11,4 +11,6 @@ General CoreOS Pre-Installation Notes:
 - ansible_python_interpreter should be `/opt/bin/python`. This will be laid down by the bootstrap task.
 - The default resolvconf_mode setting of `docker_dns` **does not** work for CoreOS. This is because we do not edit the systemd service file for docker on CoreOS nodes. Instead, just use the `host_resolvconf` mode. It should work out of the box.
 
+If you use standard CoreOS image you should set `ansible_become` flag to correctly provisioning nodes.  
```

Comment:
This applies to all OSes in general. I don't see why this CoreOS distinction is required.

Reply:
I meet this problem only during used coreos. Anyway I move this to the README.